Clients/Xamarin.Interactive.Client/Preferences/Prefs.cs

```diff
@@ -42,6 +42,9 @@ public static class Theme
                 public static readonly Preference<bool> UseHighContrast
                     = new Preference<bool> ("ui.theme.useHighContrast", false);
             }
+
+            public static readonly Preference<AgentType> LastUsedAgentType
+                = new Preference<AgentType> ("ui.lastUsedAgentType", AgentType.Unknown);
```

Comment:
I'm wondering if this pref should instead be the workbook app identifier ("console-netcore" for example). Since we are presumably getting rid of `AgentType` when it becomes possible for new workbook apps/types to be added dynamically.

Additionally, does anyone think we should be storing the state of optional features (XF being the only example currently)?

Reply:
Good point--I keep forgetting that `AgentType` is going away. Not sure about optional features.